native/explorer/src/dataframe/io.rs

```diff
@@ -77,7 +77,7 @@ pub fn df_from_csv(
         .with_encoding(encoding)
         .with_columns(column_names)
         .with_dtypes(schema)
-        .with_null_values(Some(NullValues::AllColumns(vec![null_char])))
+        .with_null_values(null_char.map(|x| NullValues::AllColumns(vec![x])))
```

Comment:
Nice :+1: 

Since we can have a vec here, we can give the option to pass more than one string. WDYT?

Reply:
Agreed f9b34bba148dc710fdf6f5db024e4b54401a33de